primer/brand
# Update side-by-side recipe with sign in banner

## Summary

Adds EMU sign in banner to side by side banner, to best-effort match the latest designs.

🔗  [See it here](https://primer-453d79ad74-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-1--level-one-side-by-side)

## List of notable changes:



- adds banner to recipe
- removes automatic horizontal alignment between page heading and first input, as the banner now takes up too much vertical space to make this feasible.

## What should reviewers focus on?



- Review the design and storybook and check we're happy with the outcome

## Steps to test:



1. [Open this storybook story](https://primer-453d79ad74-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-1--level-one-side-by-side) to see the default form version
1. [Open this storybook story](https://primer-453d79ad74-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-1--level-one-side-by-side&args=formType:extended) to see the extended, Enterprise version of the form.


## Supporting resources (related issues, external links, etc):

- [Design](https://www.figma.com/file/GVJXHPMQfjgbVfuPHXBcnr/Road-to-Universe-'23?type=design&node-id=41-16452&mode=design&t=ZK9PR3NUNjCQYIfa-4)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Design</th> <th> Storybook </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-10-23 at 11 42 36](https://github.com/primer/brand/assets/13340707/13162365-d59a-4f83-a2b0-24d00fa92960)


 </td>
<td valign="top">


![Screenshot 2023-10-23 at 11 43 28](https://github.com/primer/brand/assets/13340707/ee3eee95-5b68-434f-b69b-4e13628b3b43)

</td>
</tr>
</table>
